app.js

```diff
@@ -0,0 +1,87 @@
+import express from 'express';
+import logger from 'morgan';
+import bodyParser from 'body-parser';
+import models from './server/models/index';
+
+// Set up the express app
+const app = express();
+
+// Log requests to the console.
+app.use(logger('dev'));
+
+// Parse incoming requests data (https://github.com/expressjs/body-parser)
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+
+// Setup a default catch-all route that sends back a welcome message in JSON format.
+app.get('/', (req, res) => res.status(200).send({
```

Comment:
I will like you to make the message here dynamic. It should show two things:

1. The url
2. The current date

Something like `blabla.com was visited on 02/01/2018`

You can check https://stackoverflow.com/questions/7077770/window-location-href-and-window-open-methods-in-javascript for how to check the current url 

AND

https://docs.microsoft.com/en-us/scripting/javascript/reference/date-now-function-javascript on how to check the current date and time.

Reply:
I tried using const url = window.location.href; but i keep getting window is undefined error